spec/fixtures/approvals/publish_contract_nothing_exists_with_webhook.approved.json

```diff
@@ -89,7 +89,7 @@
         },
         {
           "type": "debug",
-          "text": "  Webhook \"foo webhook\" triggered for event contract_content_changed.\n    translation missing: en.pact_broker.messages.webhooks.triggered_webhook_see_notices"
+          "text": "  Webhook \"foo webhook\" triggered for event contract_content_changed.\n    View logs at http://example.org/triggered-webhooks/1234/logs"
```

Comment:
Ran `bundle exec approvals verify` and approved the `diff`.
Looks like the `en.pact_broker.messages.webhooks.triggered_webhook_see_notices`
translation has been introduced in df899ebd3 hence causing the spec to fail?